IBAnimatable/FlipAnimator.swift

```diff
@@ -173,7 +173,7 @@ private extension FlipAnimator {
                              completion: @escaping AnimatableCompletion) {
     UIView.animateKeyframes(withDuration: transitionDuration, delay: 0, options: .layoutSubviews, animations: {
       UIView.addKeyframe(withRelativeStartTime: 0.0, relativeDuration: 0.5, animations: {
-        flippedSectionOfFromView.0.layer.transform = self.rotate(angle: self.reverse ? -.pi * 2 : .pi * 2)
+        flippedSectionOfFromView.0.layer.transform = self.rotate(angle: (self.reverse ? -.pi : .pi) / 2)
```

Comment:
Yeah we do but #446 removes all of them where not necessary. I could've missed some but I think I got them all 😛